src/commands/moderations/WarningsCommand.ts

```diff
@@ -8,16 +8,18 @@ export default class WarningsCommand extends BaseCommand {
 	constructor() {
 		super("warnings", "moderation");
 	}
+
 	async run(client: DiscordClient, interaction: CommandInteraction) {
 		const member = interaction.options.data[0].member as GuildMember;
-		const warnings = (await getWarnings(member.user.id)) as WarningData[];
+		const warnings = await getWarnings(member.user.id) as WarningData[];
```

Comment:
the "unnecessary casting" is what my prettier does and I'm actually happy with it ;-;